src/main/resources/application.yml

```diff
@@ -10,6 +10,9 @@ queueconfig:
   action-field-queue: Action.Field
   outbound-exchange: adapter-outbound-exchange
   adapter-outbound-queue: RM.Field
+  case-event-exchange: events
+  refusal-queue: FieldworkAdapter.Refusals
```

Comment:
do you think we would have separate queues for refusals and receipting etc in the future? or just one queue bound to all those events?